spark-operator/src/main/java/org/apache/spark/k8s/operator/config/SparkOperatorConf.java

```diff
@@ -334,6 +334,30 @@ public final class SparkOperatorConf {
           .defaultValue(19090)
           .build();
 
+  public static final ConfigOption<Boolean> ENABLE_PROMETHEUS_TEXT_BASED_FORMAT =
+      ConfigOption.<Boolean>builder()
+          .key("spark.kubernetes.operator.metrics.enablePrometheusTextBasedFormat")
```

Comment:
For new configuration, we had better rename `enableXXX` to `XXXEnabled` for consistency.
```
- spark.kubernetes.operator.metrics.enablePrometheusTextBasedFormat
+ spark.kubernetes.operator.metrics.PrometheusTextBasedFormatEnabled
```

Reply:
Thanks @dongjoon-hyun for the details, I'm ok with the config names suggested by you.